src/Symfony/Component/DomCrawler/FormFieldRegistry.php

```diff
@@ -124,13 +124,15 @@ public function has($name)
     public function set($name, $value)
     {
         $target = &$this->get($name);
-        if (!is_array($value) || $target instanceof Field\ChoiceFormField) {
+        if ($target instanceof Field\FormField) {
```

Comment:
Will this even work when `$value` is an array and `$target` is no `ChoiceFormField`?